tests/100_installation.robot

```diff
@@ -6,29 +6,30 @@ Library             SeleniumLibrary
 Resource            ../resources/keywords/import_provider_account.resource
 
 Suite Setup         Run Keywords
-...                 Set Library Search Order    SeleniumLibrary
-...                 AND    Skip If    ${DBaaSPolicyEnabled}
+...                     Set Library Search Order    SeleniumLibrary
+...                     AND    Skip If    ${DBaaSPolicyEnabled}
```

Comment:
What is this If for?

Reply:
@ikanias The scope of this PR is to align the existing with the spec on robotidy.yaml.
But, I could recall this "if" statement is skip the execution of entire test suite if the ${DBaaSPolicyEnabled} is set to "True".